homeassistant/components/google_assistant/helpers.py

```diff
@@ -101,10 +104,15 @@ def should_2fa(self, state):
         # pylint: disable=no-self-use
         return True
 
-    async def async_report_state(self, message):
+    async def async_report_state(self, message, agent_user_id: str):
         """Send a state report to Google."""
         raise NotImplementedError
 
+    async def async_report_state_all(self, message):
+        """Send a state report to Google for all previously synced users."""
+        for agent_user_id in self._agent_user_ids:
+            await self.async_report_state(message, agent_user_id)
```

Comment:
Must this be run sequentially?

Reply:
no probably not. i'd expect google servers to support parallel requests. good idea.